Functions.Templates/Templates/SqlTrigger-CSharp-Isolated/.template.config/template.json

```
@@ -0,0 +1,71 @@
+{
+    "author": "Microsoft",
+    "classifications": [
+        "Azure Function",
+        "Trigger",
+        "SQL"
+    ],
+    "name": "SqlTriggerBindingIsolated",
+    "identity": "Azure.Function.CSharp.Isolated.SqlTriggerBinding.2.x",
```

This should be 3.x, right? (I assume this is supposed to match the version)

```suggestion
    "identity": "Azure.Function.CSharp.Isolated.SqlTriggerBinding.3.x",
```

---

This version thing probably isn't even needed. @soninaren Can you please confirm and I can update accoridingly. Thanks.

---

@fabiocav Can you please confirm if the version is needed.

---

Not needed. You just need to make sure this is a unique identifier.